luaui/Widgets/gui_awards.lua

```diff
@@ -10,53 +14,146 @@ function widget:GetInfo()
 	}
 end
 
-local glCallList = gl.CallList
-
-local thisAward
-
-local widgetScale = 1
-
-local drawAwards = false
-local centerX, centerY -- coords for center of screen
-local widgetX, widgetY -- coords for top left hand corner of box
-local width = 880
-local height = 520
-local widgetWidthScaled = math.floor(width * widgetScale)
-local widgetHeightScaled = math.floor(height * widgetScale)
-local quitRightX = math.floor(100 * widgetScale)
-local graphsRightX = math.floor(250 * widgetScale)
-local closeRightX = math.floor(30 * widgetScale)
-
-local Background
-local FirstAward, SecondAward, ThirdAward, FourthAward
-local threshold = 150000
-local CowAward
-local OtherAwards
-
-local chobbyLoaded = (Spring.GetMenuName and string.find(string.lower(Spring.GetMenuName()), 'chobby') ~= nil)
-
-local white = "\255" .. string.char(251) .. string.char(251) .. string.char(251)
-
-local playerListByTeam = {} -- does not contain specs
-
-local fontfile = "fonts/" .. Spring.GetConfigString("bar_font", "Poppins-Regular.otf")
-local fontfile2 = "fonts/" .. Spring.GetConfigString("bar_font2", "Exo2-SemiBold.otf")
-local font, font2, titleFont
-
-local viewScreenX, viewScreenY = Spring.GetViewGeometry()
-
-local UiElement
-
-local function colourNames(teamID)
+local document
+widget.rmlContext = nil
+
+local playerListByTeam = {}
+local isVisible = false
+local dm_handle
```

Comment:
Since this variable is going to appear in pretty much every rmlui widget, can we standardize on a friendlier name.
`documentHandle`
`modelHandle`
`documentModelHandle`